integration-tests/tests/exit_syscall_payload.rs

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! This crate contains integration tests, including the exit syscall payload test.
+
+#![deny(clippy::all)]
+#![deny(missing_docs)]
+
+use std::process::Command;
+use std::time::Duration;
+use wait_timeout::ChildExt;
+
+mod common;
+
+/// This test runs the payload in the SGX keep using the SGX shim.
+#[cfg_attr(not(has_sgx), ignore)]
+#[test]
+fn sys_exit() {
+    // Define directories
+    let setup = common::Setup::new();
+    // Find the current payload
+    let payload = common::find_filepath(setup.payload_dir, "test_sys_exit").unwrap();
+    // Run the test
+    let mut payload_test = Command::new(setup.keep)
+        .current_dir(setup.wksp_root)
+        .arg("--code")
+        .arg(payload)
+        .arg("--shim")
+        .arg(setup.shim)
+        .spawn()
+        .expect("failed to run exit payload test");
+
+    let ecode = match payload_test.wait_timeout(Duration::from_secs(60)).unwrap() {
+        Some(status) => status,
+        None => {
+            payload_test.kill().unwrap();
+            payload_test.wait().unwrap()
+        }
+    };
+    assert!(ecode.success());
+}
+
+#[cfg_attr(not(has_sgx), ignore)]
+#[test]
+#[should_panic]
+fn timeout_sys_exit() {
+    // Define directories
+    let setup = common::Setup::new();
+    // Find the current payload
+    let payload = common::find_filepath(setup.payload_dir, "timeout_test_sys_exit").unwrap();
+    // Run the test
+    let mut payload_test = Command::new(setup.keep)
+        .current_dir(setup.wksp_root)
+        .arg("--code")
+        .arg(payload)
+        .arg("--shim")
+        .arg(setup.shim)
+        .spawn()
+        .expect("failed to run exit payload test");
+
+    let ecode = match payload_test.wait_timeout(Duration::from_secs(60)).unwrap() {
```

Comment:
Also, for testing the watchdog/autokill functionality, I think a duration of 1-5 seconds is probably sufficient. Give the keep enough time to setup and enter the loop for a little bit. This would hold up our CI for another entire minute.